misc/language-tour-guide/src/content/1-basic-types-and-operations/strings2.mdx

```diff
@@ -0,0 +1,14 @@
+export const frontmatter = { order: 19 };
+
+# Strings
+
+Strings in Elixir are represented internally by contiguous sequences of bytes, known as binaries. Thus, we can use functions such as `byte_size` on strings. Functions specific to strings are in the `String` module.
```

Comment:
You have omitted the UTF-8 and the difference between `byte_size` and `String.length`. was that intentional?